app/src/main/java/com/YC2010/jason/myclass/ui/fragments/SearchFragment.java

```diff
@@ -38,7 +45,39 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         if (mView == null) {
             mView = inflater.inflate(R.layout.fragment_search, container, false);
+
+            if (!Connections.isNetworkAvailable(mActivity)) {
+                LinearLayout errorView = (LinearLayout) mView.findViewById(R.id.error_view);
+                TextView errorMessage = (TextView) mView.findViewById(R.id.error_message);
+                ImageView errorIcon = (ImageView) mView.findViewById(R.id.error_icon);
+
```

Comment:
if you are displaying the retry button, better implement it. or we can hide it and implement it later.(open an issue for it, nice to have feature)